MediaBrowser.Model/Dlna/StreamInfo.cs

```diff
@@ -108,8 +108,10 @@ public StreamInfo()
         public string MediaSourceId => MediaSource?.Id;
 
         public bool IsDirectStream =>
-            PlayMethod == PlayMethod.DirectStream ||
-            PlayMethod == PlayMethod.DirectPlay;
+            !(MediaSource?.VideoType == VideoType.Dvd
+                || MediaSource?.VideoType == VideoType.BluRay)
+            && (PlayMethod == PlayMethod.DirectStream
+                || PlayMethod == PlayMethod.DirectPlay);
```

Comment:
```suggestion
            MediaSource?.VideoType is not (VideoType.Dvd or VideoType.BluRay)
            && PlayMethod is PlayMethod.DirectStream or PlayMethod.DirectPlay;
```

I find this more legible